spec/queries/articles/active_threads_query_spec.rb

```diff
@@ -11,15 +11,15 @@
         article = create(:article, tags: "discuss", score: described_class::MINIMUM_SCORE + 1)
         create(:article, published_at: 2.days.ago, tags: "discuss", score: described_class::MINIMUM_SCORE + 1)
 
-        result = described_class.call(options: { tags: "discuss", time_ago: "latest", count: 10 })
+        result = described_class.call(tags: "discuss", time_ago: "latest", count: 10)
```

Comment:
Likewise, we no longer need `options{...}` in the spec either (same goes for line `22`).